src/test/java/uk/gov/hmcts/sptribs/document/bundling/BundlingServiceTest.java

```diff
@@ -262,37 +292,260 @@ void shouldReturnNullWhenNoBundles() {
 
         when(bundlingClient.createBundle(any(), any(), any())).thenReturn(bundleResponse);
 
-        Callback callback = new Callback(updatedCaseDetails, beforeDetails, CREATE_BUNDLE, true);
-        BundleCallback bundleCallback = new BundleCallback(callback);
-        //When
-        List<Bundle> result = bundlingService.createBundle(bundleCallback);
-        List<ListValue<Bundle>> resultList = bundlingService.buildBundleListValues(result);
+        final Callback callback = new Callback(updatedCaseDetails, beforeCaseDetails, CREATE_BUNDLE, true);
+        final BundleCallback bundleCallback = new BundleCallback(callback);
+
+        final List<Bundle> result = bundlingService.createBundle(bundleCallback);
+        final List<ListValue<Bundle>> resultList = bundlingService.buildBundleListValues(result);
 
-        //Then
         verify(bundlingClient).createBundle(any(), any(), any());
         assertThat(result).isEmpty();
         assertThat(resultList).isNull();
-    }*/
+    }
 
     @Test
     void shouldReturnNullWhenNoBundleFolders() {
-        //Given
-        List<BundleFolder> bundleFolderList = Collections.emptyList();
-        //When
-        List<ListValue<BundleFolder>> resultList = bundlingService.buildBundleFolderListValues(bundleFolderList);
-        //Then
+        final List<BundleFolder> bundleFolderList = Collections.emptyList();
+        final List<ListValue<BundleFolder>> resultList = bundlingService.buildBundleFolderListValues(bundleFolderList);
         assertThat(resultList).isNull();
     }
 
     @Test
     void shouldReturnNullWhenNoBundleDocuments() {
```

Comment:
A new test case can be added to create bundled document with values.

Reply:
I think these are covered in the parametrised tests `shouldCreateBundleListValues` and `shouldCreateBundle`

Reply:
reply is https://github.com/hmcts/sptribs-case-api/pull/1677#discussion_r1487565848